### PR TITLE
Add support for DSM7

### PR DESCRIPTION
**Pull request by maisem**
- Move all of the tailscale output files on DSM6 to `/var/packages/Tailscale/target/var`, and on DSM7 to `/var/packages/Tailscale/var`.
- Rename the output spks to include the DSM version
- Add `Tailscale.sc` to document the ports used for the Synology firewall
- Run `tailscaled` as non-root on DSM7

Updates tailscale/tailscale#48, #36

Signed-off-by: Maisem Ali <maisem@tailscale.com>

### CODE REVIEW

File: src/INFO.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ if [ "$DSM_VERSION" = "6" ]; then
   os_max_ver="7.0-40000"
 else
   os_min_ver="7.0-40000"
-  os_max_ver=""
+  os_max_ver="8.0-00000"
```
Comment (crawshaw) [inline src/INFO.sh:40]: I believe we were told to leave this empty on dsm 7?

**Review by crawshaw — APPROVED** [review]

File: src/INFO.sh (hunk 1 above)
Comment (maisem) [reply src/INFO.sh]: removed and verified.

**Comment by kjacobsen** [comment]
Great work, any idea when the new package will be available?

**Comment by DentonGentry** [comment]
It has been sent to Synology for review. I would expect a round of feedback or two, with fixes and another submission for review.

The timing is mostly not under our control, I would expect it to be weeks not days.

**Comment by bilke** [comment]
I manually created the package and installed it on DSM 7. When running `sudo tailscale up`:

```
can't fetch status from tailscaled: Get "http://local-tailscaled.sock/localapi/v0/status": dial unix /var/run/tailscale/tailscaled.sock: connect: no such file or directory
```

When checking tailscaled it seems that is uses another port:

```
$ ps aux | grep tailsca
tailsca+ 12810  1.1  0.1 717148 15420 ?        Sl   11:16   0:03 /volume2/@appstore/Tailscale/bin/tailscaled --state=/volume2/@appdata/Tailscale/tailscaled.state --socket=/volume2/@appdata/Tailscale/tailscaled.sock --port=41641 --tun=userspace-networking
```

Is there a way to provide the socket when running `tailscale up`?

**Comment by kudos** [comment]
@bilke I used this command to get it up and running. For some reason the binary didn't get added to my path either.
`sudo /volume1/@appstore/Tailscale/bin/tailscale --socket=/volume1/@appdata/Tailscale/tailscaled.sock up`

**Pull request merged**